API-Spec.md

```diff
@@ -0,0 +1,104 @@
+# ToDo API
+---
+This is a blueprint of the ToDo API. The API showcases POST, GET, PUT, DEL
+
+## Endpoints
+---
+
+### Create ToDo's [POST]
+Create a Todo
+
+URL: `POST/ToDos/`
```

Comment:
the `HTTP Verb` is part of the URL?

Generally the format should be

`VERB /urlToEndpoint`